.github/workflows/soaking.yml

```diff
@@ -251,7 +253,7 @@ jobs:
             -a ${{ matrix.architecture }}
       - name: Set output if layer Soak Tests has error
         id: set-layer-if-error-output
-        if: ${{ failure() }}
+        if: ${{ success() }}
```

Comment:
Where is the official documentation for `success()` existing? I have found the docs for [failure()](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-using-status-check-functions). 

Also, can you expand why this change is necessary? From what I can see this step only needs to happen if a previous step fails, not if it is successful.

Reply:
yeah added the description for it.

Reply:
Why would we want to run `echo "::set-output name=${{ matrix.language }}-${{ matrix.instrumentation-type }}-error::FAILED"` if the previous steps were successful?

Reply:
Think i could improve this PR.